### PR TITLE
First addition of visdag content.

### PR DESCRIPTION
**Pull request by otherorb**
Submitting the first pull request to origin.

### CODE REVIEW

File: .telemetry/id.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+instance_id: b4217c06-71cb-4406-8e5e-f9652868b855
```
Comment (rbeyer) [inline .telemetry/id.yaml]: This looks ephemeral, should it be in the repo?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,132 @@
-# visds
-VIPER VIS Data System
+<<<<<<< Updated upstream
```
Comment (rbeyer) [inline README.md]: There are conflict markers in this file, please clear.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<<<<<<< Updated upstream
```
Comment (rbeyer) [inline setup.py]: There are conflict markers in this file, please clear.

File: visdag/ops/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (rbeyer) [inline visdag/ops/__init__.py]: This file should exist and be empty or have contents, but a single blank line is weird.

File: visdag/schedules/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+
```
Comment (rbeyer) [inline visdag/schedules/__init__.py]: blank line

File: visdag/visds_config.yaml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+solids:
+    visds_solid:
+      db_config:
+        - DB_TYPE=postgresql
+        - POSTGRES_USER=viper
+        - POSTGRES_PASSWORD=viper
```
Comment (rbeyer) [inline visdag/visds_config.yaml]: This file should not be in the repo.  You could include a visdag/visds_config.example.yml in the repo to show people what is expected, but the install instructions should instruct a user to create their own visdag/visds_config.yml from the example file.

File: visdag/watcher.json [hunk 7]
```diff
@@ -0,0 +1,262 @@
+[
```
Comment (rbeyer) [inline visdag/watcher.json]: Why is this file in the repo?

File: visdag_tests/__init__.py [hunk 8]
```diff
@@ -0,0 +1 @@
+
```
Comment (rbeyer) [inline visdag_tests/__init__.py]: Blank line.

File: visdag_tests/test_assets.py [hunk 9]
```diff
@@ -0,0 +1 @@
+
```
Comment (rbeyer) [inline visdag_tests/test_assets.py]: Blank line.

**Review by rbeyer — CHANGES_REQUESTED** [review]
Also seems like a lot of redundant versions of watch_cam.

File: .telemetry/id.yaml (hunk 1 above)
Comment (otherorb) [reply .telemetry/id.yaml]: Removed.

File: visdag/ops/__init__.py (hunk 4 above)
Comment (otherorb) [reply visdag/ops/__init__.py]: Emptied file.

File: visdag/watcher.json (hunk 7 above)
Comment (otherorb) [reply visdag/watcher.json]: Junk file left over from a git add on the directory. Removed.

File: visdag_tests/__init__.py (hunk 8 above)
Comment (otherorb) [reply visdag_tests/__init__.py]: Okay, this is weird. I didn't touch any of the files in visdag_tests, so it looks like dagster is creating these __init__.py files with an empty line. Not sure why. Anyway, that's fixed.

File: visdag/schedules/__init__.py (hunk 5 above)
Comment (otherorb) [reply visdag/schedules/__init__.py]: Fixed.

File: visdag/visds_config.yaml (hunk 6 above)
Comment (otherorb) [reply visdag/visds_config.yaml]: Missed this file when changing to .env configuration. Removed file. 

A TODO is to update the install instructions. I want to get this PR done first, so will not add a new file yet.

File: visdag_tests/test_assets.py (hunk 9 above)
Comment (otherorb) [reply visdag_tests/test_assets.py]: Fixed.

File: README.md (hunk 2 above)
Comment (otherorb) [reply README.md]: Cleared.

File: setup.py (hunk 3 above)
Comment (otherorb) [reply setup.py]: Cleared.

**Comment by otherorb** [comment]
Note that I removed the watchcam files for now. The one useful version, rewritten for use as an asset_sensor and will be added back in with the new asset_sensor branch.

**Review by rbeyer — APPROVED** [review]

**Pull request merged**